tasks/basic-sanity-check.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: basic-sanity-check
```

Comment:
Should we rename the task and the file to sanity-label-check? Since we plan on adding separate tasks for sanity-deprecated-image check and sanity-signed-rpm-check, etc. ?

Reply:
Or the task could be just `conftest` and as parameter it could take the policy. And then you can have pipelines which will be composed of multiple tasks.

BTW there will be test pipelines or it will be part of other pipelines (eg. build, release)?

Reply:
> Should we rename the task and the file.
That would make sense, since we are checking for labels.

> Or the task could be just conftest and as parameter it could take the policy. And then you can have pipelines which will be composed of multiple tasks.
This aproach is also possible, I would say we can keep this as a task for now and include it in a pipeline if it turns out that we want to have such approach.

> BTW there will be test pipelines or it will be part of other pipelines (eg. build, release)?
This should be part of other pipeline(s), currently I don't know about having standalone test pipeline.